### ISSUE TITLE
Successfully completes Recreating Database but no files are restored

### ISSUE THREAD

**Issue by Michaelvvt** [duplicati/duplicati#3669]
- [x] I have searched open and closed issues for duplicates.

----------------------------------------

## Environment info

- **Duplicati version**: 2.0.4.5_beta_2018-11-28
- **Operating system**: Unraid Docker Container
- **Backend**: Google Drive / Local

## Description

I'm trying to recover 50gb backup worth of data from a full server backup. After starting the complete restore and waiting 19 hours for 'Recreating database...' to take place it simply says Repair complete in the logs, however, no files are actually being restored. I tried to restore again, thinking that the repair had worked, however the same problem happened. I then downloaded all of the files from Google Drive onto a local cache drive and attempted to recover twice locally, with the same result. I'm lost as to why no files are actually being recovered, however, it seems to run this Recreating Database repair each time.

## Steps to reproduce

1. Restore a complete backup from Google Drive
2. 'Recreating Database...'
3. Repair Complete
4. No files are actually restored

- **Actual result**:
Progress bar goes back to default 'next backup at timestamp' and no files are recovered
- **Expected result**:
Recovering Files

## Screenshots

![](https://i.gyazo.com/a943c575b4133c4104d004bce70ed547.png)

**Comment by warwickmm** [comment]
Is the `tmp` folder in the docker container mapped to a share in your UnRAID array?  Can you see a bunch of temporary files appearing during the restore process?

**Comment by Michaelvvt** [comment]
No sorry, it was mapped in the linux root. I checked and there are files there. After many runs, it retrieved the folder structure but did not recover any files. I've since moved the temp folder onto the cache drive map and I can see the temp files fluctuating. The restore process is going much slower now so I assume this is because it's actually working. It has taken two days so far and seems to still have a lot to go. I'll keep you updated, thank you for your initial response.
